src/async.h

```diff
@@ -6,7 +6,8 @@
 
 class SetPasswordWorker : public Nan::AsyncWorker {
   public:
-    SetPasswordWorker(const std::string& service, const std::string& account, const std::string& password,
+    SetPasswordWorker(const std::string& service, const std::string& account, const std::string& password, const std::string& targetname,
+                       const int credType, const int credPersist,
```

Comment:
make them an enum and expose in index.d.ts

Reply:
I made them an enum. I can static cast to DWORD it in async.cc as opposed to here if thats a better location.
The reason we cant type cast it on the way in is that mac will not support DWORD. Do let me know your thoughts.